src/pages/ShowBlogDetails/ShowBlogDetails.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect } from "react";
+import { useParams } from "react-router";
+import styles from "./ShowBlogDetails.module.less";
+import Loader from "../../Common-Components/Loader/Loader";
+
+function ShowBlogDetails(props) {
+  const id = useParams();
+
+  const { id: blogId } = id;
```

Comment:
can be destructured in line 7 only like: const { id: blogId } = useParams()

Reply:
Done